main/input_default.h

```diff
@@ -107,6 +107,19 @@ class InputDefault : public Input {
 	Map<int, Joystick> joy_names;
 	int fallback_mapping;
 	RES custom_cursor;
+
+	struct Tracker {
+		TrackerType	type;
+		StringName	name;
+		Transform		transform;
```

Comment:
I think you should singlespace those, not align them with tabs or something else.